src/templates/landing_pages/ico.html.tt

```diff
@@ -685,10 +705,23 @@
                     </div>
                     <div class="gr-12">
                         <div class="faq">
+                            <h4>[% l("What will be the token symbol?") %]</h4>
+                            <p>[% l("We intend to apply for the token symbol BINARY.") %]</p>
+                        </div>
+                    </div>
+                    <div class="gr-12">
+                        <div class="faq">
                             <h4>[% l("What are the benefits of purchasing tokens?") %]</h4>
-                            <p>[% l("There will be two main benefits of purchasing tokens.") %]</p>
+                            <p>[% l("There will be three main benefits of purchasing tokens.") %]</p>
                             <p>[% l("First, token holders will receive dividend-linked payments equivalent to the dividends paid by the Company on its Ordinary Shares. Therefore, token holders will share in the future success of the Company in the same way as do shareholders.") %]</p>
-                            <p>[% l("Second, token holders will be able to freely trade their tokens. In contrast, Ordinary Shares are not listed and therefore cannot be traded on any exchange. In that sense, the tokens will be a superior investment to Ordinary Shares.") %]</p>
+                            <p>[% l("Second, tokens may be converted into Ordinary Shares of the Company (subject to certain conditions, as described in the Information Memorandum), thereby providing the legal link between tokens and Ordinary Shares.") %]</p>
+                            <p>[% l("Third, token holders will be able to freely trade their tokens. In contrast, Ordinary Shares are not listed and therefore cannot be traded on any exchange. In that sense, the tokens will be a superior investment to Ordinary Shares.") %]</p>
+                        </div>
+                    </div>
+                    <div class="gr-12">
+                        <div class="faq">
+                            <h4>[% l("Where can I view the source code of the tokens?") %]</h4>
+                            <p>[% l("Binary.com intends to deploy the Mini Me token. Please view our blog post detailing our reasons for choosing this token.") %]</p>
```

Comment:
"Mini Me" should be a hyperlink to https://github.com/Giveth/minime (in a new tab)